nuxeo-core/nuxeo-core-api/src/test/java/org/nuxeo/ecm/core/DummyLoginAs.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * (C) Copyright 2018 Nuxeo SA (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.nuxeo.ecm.core;
+
+import java.security.Principal;
+
+import javax.security.auth.login.LoginContext;
+import javax.security.auth.login.LoginException;
+
+import org.nuxeo.ecm.core.api.impl.UserPrincipal;
+import org.nuxeo.ecm.core.api.local.ClientLoginModule;
+import org.nuxeo.ecm.core.api.security.SecurityConstants;
+import org.nuxeo.runtime.api.login.LoginAs;
+
+/**
+ * Computation rely on Framework.loginAsUser(String username).
+ * <p>
+ * This Framework wrapper method relies on the LoginAs service, thus we have to provide an implementation that allows
+ * tests to be executed.
+ *
+ * @since HF
```

Comment:
Is that the convention?

Reply:
Indeed no, will put 11.1 as this class exists on 11.1.